stdlib/2/urllib2.pyi

```diff
@@ -45,14 +52,21 @@ class Request(object):
     def get_header(self, header_name: str, default: Optional[str] = ...) -> str: ...
     def header_items(self): ...
 
+class _Response(_SupportsRead, metaclass=ABCMeta):
```

Comment:
Would this need to have more methods now that were removed from `_SupportsRead`? What methods is this object documented to support?